snap/snapcraft.yaml

```diff
@@ -160,35 +160,44 @@ parts:
     plugin: autotools
     source: https://git.launchpad.net/ubuntu/+source/network-manager
     source-type: git
-    source-branch: applied/ubuntu/jammy-updates
+    source-branch: applied/ubuntu/noble
     build-packages:
-      - intltool
+      - gettext
       - libglib2.0-dev
       - ppp-dev
+      - libselinux1-dev
+      - libaudit-dev
       - libgnutls28-dev
       - uuid-dev
-      - systemd
+      - systemd-dev
       - libsystemd-dev
       - libudev-dev
       - libgirepository1.0-dev
       - gobject-introspection
       - python3-gi
       - libpsl-dev
       - libcurl4-gnutls-dev
+      - gtk-doc-tools
```

Comment:
We are disabling gtkdoc, is this needed?

Reply:
Yep it's still needed for the build
`./autogen.sh: 25: gtkdocize: not found`